src/js/hooks/invoice/useInvoicePrepaidItemsTable.jsx

```diff
@@ -133,7 +133,11 @@ const useInvoicePrepaidItemsTable = ({
 
   // validation for order adjustments
   const validateUnitPrice = (row) => {
-    if (row?.unitPrice === 0 || Math.abs(row?.unitPrice) > Math.abs(row?.amount)) {
+    const amountAvailableToInvoice =
+      Math.abs(row?.amountAvailableToInvoice) + Math.abs(row?.amount);
```

Comment:
why plus?

Reply:
I think this is a case where we have "amount available **outside** this invoice item" **plus** "amount **from this** item" to determine full available amount

Reply:
I just thought it'd be amount available to invoice = current amount available to invoice - amount we're invoicing for in this row